ttnn/cpp/pybind11/pytensor.cpp

```diff
@@ -446,7 +440,13 @@ Tensor convert_python_tensors_to_tt_tensors(
     std::vector<Tensor> tt_shards;
     for (const auto& shard : tensor_shards) {
         tt_shards.push_back(detail::convert_python_tensor_to_tt_tensor(
-            shard, data_type, Layout::ROW_MAJOR, tile, MemoryConfig{}, nullptr, true));
+            shard,
+            data_type,
+            /*optional_layout=*/std::nullopt,
```

Comment:
Note on this change: empty `optional_layout` falls back to the "right" layout, either tile (for block float types), or row major (for the rest). So this behavior hasn't changed.